async_fuse/src/main.rs

```diff
@@ -1,3 +1,29 @@
+#![deny(
+    // The following are allowed by default lints according to
+    // https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html
+    anonymous_parameters,
+    bare_trait_objects,
+    // box_pointers, TODO: fix box pointers
```

Comment:
What's following plan for these TODO?

Reply:
some are easy to fix, such as unreachable_pub, some need more work, such as unused_results, unsafe_code, missing_docs.

I'll open some issues to track them.